version-changelog.js

```diff
@@ -0,0 +1,96 @@
+#! /usr/bin/env node
+
+var readline = require('readline');
+var fs = require('fs');
+var childProcess = require('child_process');
+
+var rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+var pluginName = process.argv[2];
+var pluginSlug = pluginName.substr(3);
+var mainPHPFile = pluginName + '/' + pluginName + '.php';
```

Comment:
I think we need to add the "-db.php" file here as well for the version change, right?